apps/elf-council-frontend/src/ui/proposals/ProposalsPage.tsx

```diff
@@ -53,17 +52,14 @@ export default function ProposalsPage({
   const isTailwindSmallScreen = useIsTailwindSmallScreen();
   const isTailwindLargeScreen = useIsTailwindLargeScreen();
 
-  const unverifiedProposals = useUnverifiedProposals(proposalsJson.proposals);
```

Comment:
Ah I think @sentilesdal was requesting we keep this part in order to stay in sync with the on-chain proposals at all times in the UI. 

Otherwise there is still a window of time where an attacker could have an on-chain proposal that doesn't show in the UI.

(It's still good to scrape and annotate them in the JSON though as you've done here! 🤘)

Reply:
Ahh okay, thanks for clarification! Makes sense to keep this in